pkg/registry/apis/provisioning/jobs/exporter.go

```diff
@@ -39,13 +42,21 @@ func NewExporter(
 ) (Exporter, error) {
 	dynamicClient := parser.Client()
 	folders := dynamicClient.Resource(schema.GroupVersionResource{
-		Group:    "folder.grafana.app",
-		Version:  "v0alpha1",
-		Resource: "folders",
+		Group:    folders.GROUP,
+		Version:  folders.VERSION,
+		Resource: folders.RESOURCE,
 	})
+
+	dashboards := dynamicClient.Resource(schema.GroupVersionResource{
+		Group:    "dashboard.grafana.app",
+		Version:  "v2alpha1",
```

Comment:
```suggestion
		Version:  "v1alpha1",
```
the v1 flavor will be stable -- v2 is all different, so lets avoid it for now